assets/chat/js/chat.js

```diff
@@ -2163,182 +2167,76 @@ class Chat {
   cmdEMBED(parts) {
```

Comment:
I think having separate error messages related to the number of arguments is unnecessary. Try to simplify all this such that you just print two things on invalid use of the command: the error from the convertor, and a usage hint.